marlowe-dashboard-client/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "@tailwindcss/forms": "^0.3.2",
+    "@webpack-cli/serve": "^1.5.1",
```

Comment:
What's this for?

Reply:
No idea, it kind of happened automatically when running webpack (or something that webpack asked me to do)